### PR TITLE
ci(jenkins): enable slack notifications

### PR DESCRIPTION
**Pull request by v1v**
## What

Enable slack notifications, similar to the email notifications, when a release is triggered.
Enable notifications if something bad happened with the release.

## Why

Ensure the right audience is in the loop.

## Tests

the commit https://github.com/elastic/apm-agent-rum-js/pull/908/commits/24707b292729481c5041b5a248ee8be92706992f caused:

- Slack notification

image

- Email notification

image

### CODE REVIEW

**Review by vigneshshanmugam — APPROVED** [review]
So cool, Thanks @v1v

File: Jenkinsfile [hunk 1]
```diff
@@ -326,7 +326,7 @@ pipeline {
               }
               post {
                 success {
-                  emailext subject: "[${env.REPO}] Release published", to: "${env.NOTIFY_TO}", body: "Great news, the release has been done successfully."
+                  notifyStatus(slackStatus: 'good', subject: "[${env.REPO}] Release published", body: "Great news, the release has been done successfully. (<${env.RUN_DISPLAY_URL}|Open>)")
```
Comment (jahtalab) [inline Jenkinsfile:329]: nit: Can we add a link to the release page on github as well?

**Review by jahtalab — COMMENTED** [review]
Thanks @v1v for a quick action on this. Just a minor comment.

**Review by mdelapenya — APPROVED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (v1v) [reply Jenkinsfile:329]: Do you mean https://github.com/elastic/apm-agent-rum-js/releases ?
```suggestion
                  notifyStatus(slackStatus: 'good', subject: "[${env.REPO}] Release published", body: "Great news, the release has been done successfully. (<${env.RUN_DISPLAY_URL}|Open>). \n Release URL: (<https://github.com/elastic/apm-agent-rum-js/releases|Here>)")
```

Comment (jahtalab) [reply Jenkinsfile:329]: That's correct. This is for people who might not know where to look when a release has finished.

**Review by kuisathaverat — APPROVED** [review]

**Pull request merged**